recipes/libyaml/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include("conanbuildinfo.cmake")
+conan_basic_setup()
+
+add_subdirectory("source_subfolder")
```

Comment:
Brought over from bincrafters, not sure whether it's needed...

Reply:
yes, it's required. This cmake file is a wrapper, which calls conan_basic_setup() and set all cmake definitions passed by Conan client.